src/net/download.rs

```diff
@@ -1330,10 +1380,10 @@ impl PeerNetwork {
     }
 
     /// Prioritize block requests -- ask for the rarest blocks first
-    fn prioritize_requests(requests: &HashMap<u64, VecDeque<BlockRequestKey>>) -> Vec<u64> {
+    fn prioritize_requests<T: Copy>(requests: &HashMap<T, VecDeque<BlockRequestKey>>) -> Vec<T> {
```

Comment:
Please remove the `<T: Copy>` type parameter and revert to using the `u64` as the key -- you're doing that in both cases anyway.

Reply:
Addressed with https://github.com/blockstack/stacks-blockchain/pull/2013/commits/8325ca951c8a0d9144f3818915a9a89a88062097